Website/src/rendering/src/assets/css/layout/text.css

```diff
@@ -0,0 +1,5 @@
+.rich-text {
+  p + p {
```

Comment:
I am wondering if we will run into issues when we will have `<ul>`, `<ol>`, `<img>` or other elements in between `<p>` elements. None of the ContentHub items have them at the moment but it would be a good thing to test now while we are making changes in that area.

Another solution could be:
```
.rich-text {
  @apply space-y-4;
}
```
This adds special CSS classes that add 4 units of margin between each direct children of `.rich-text`. This would cover for elements other than `<p>`.